src/families/tezos/bridge/mock.js

```diff
@@ -0,0 +1,207 @@
+// @flow
+import { BigNumber } from "bignumber.js";
+import {
+  NotEnoughBalance,
+  RecipientRequired,
+  InvalidAddress,
+  FeeTooHigh,
+  InvalidAddressBecauseDestinationIsAlsoSource,
+  NotSupportedLegacyAddress,
+  NotEnoughBalanceInParentAccount,
+  AmountRequired,
+  RecommendUndelegation,
+  RecommendSubAccountsToEmpty,
+  NotEnoughBalanceToDelegate
+} from "@ledgerhq/errors";
+import type { Transaction } from "../types";
+import type { AccountBridge, CurrencyBridge } from "../../../types";
+import { isAccountBalanceSignificant } from "../../../account";
+import {
+  scanAccounts,
+  signOperation,
+  broadcast,
+  sync,
+  isInvalidRecipient
+} from "../../../bridge/mockHelpers";
+import {
+  // fetchAllBakers,
+  // hydrateBakers,
+  // asBaker,
+  isAccountDelegating
```

Comment:
at some point, we should not import `../bakers` I think.
we need to mock that.
we'll have to ask ourself how we want to address mocking bakers too.

until we don't mock bakers, it's ok to keep that, but it's going to be a real challenge. maybe bakers idea should be integrated as part of the bridge at some point? either that, or we branch some code based on `getEnv("MOCK")` but it's less powerful (i like that mock is a bridge because you can literally simultaneously have mock account and regular account. much powerful^^ – bakers.js instead is a global thing we import and not linked to a account context. we also want it to be global somehow so it's cached globally anyway...).


cc @nukky94 @henri-ly ^ your thoughts on this too would be interesting. nothing that will happen anytime soon I think but it's good we discuss at some point.